CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 2.16.0 (Unreleased)
 
+FEATURES: 
```

Comment:
Add it to `2.15.1 (Unreleased)` release.

Reply:
updated the file as suggested.